plugins/Feedback/Feedback.php

```diff
@@ -135,7 +139,12 @@ public function showReferBanner()
         $nextReminderDate = $referReminder->getUserOption();
 
         if ($nextReminderDate === false) {
-            return true;
+            $nextReminder = Date::now()->getStartOfDay()->addDay(6 * 30)->toString('Y-m-d');
```

Comment:
@flamisz I believe the "review reminder" is showing every 90 days so there's a risk both review and refer banners and shown at same point. 

Maybe after 6 months (180 days) we might show both reminders approx on a similar day? The 90 days in a review reminder is sent from `plugins/Feedback/angularjs/feedback-popup/feedback-popup.controller.js`. I now wonder if we maybe could add 135 days to the next "review reminder" date? This way it be 45 days before the popup would be shown. If no review reminder date is set yet, then we would maybe need to set the review reminder to be in 90 days and the refer reminder to be in 135 days? 

Bit complicated just wanting to avoid them being shown to close of each other :)

Reply:
Actually I check in the code, at line 111:

```php
if ($this->getShouldPromptForFeedback()) {
...
```
This way we can't show it at the same time. 
But to make sure they don't come up close to each other (next day, etc), make sense. I'll check your recommendation and add it.